examples/experimental/dagster-airlift/dagster_airlift/in_airflow/base_asset_operator.py

```diff
@@ -162,15 +181,23 @@ def launch_runs_for_task(self, context: Context, dag_id: str, task_id: str) -> N
                 "`dagster-airlift` expects that all assets mapped to a given task exist within the same code location, so that they can be executed by the same run."
             )
 
+        partitioning_info = PartitioningInformation.from_asset_node_graphql(filtered_asset_nodes)
+        partition_key_for_run = self.get_partition_key(context, partitioning_info)
         job_identifier = _get_implicit_job_identifier(next(iter(filtered_asset_nodes)))
         asset_key_paths = [asset_node["assetKey"]["path"] for asset_node in filtered_asset_nodes]
         logger.info(f"Triggering run for {job_identifier} with assets {asset_key_paths}")
+        tags = (
+            {**self.default_dagster_run_tags(context), PARTITION_NAME_TAG: partition_key_for_run}
+            if partition_key_for_run
```

Comment:
you can check for `partitioning_info` here and then not need the variable `partition_key_for_run` and call `self.get_partition_key(` directly